### PR TITLE
[INCR-95]Run python-modernize on xblock-utils

### PR DESCRIPTION
**Pull request by cclauss**
Fixes https://openedx.atlassian.net/browse/INCR-95

__ModuleNotFoundError: No module named 'django.core.urlresolvers'__

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @cclauss! I've created [OSPR-3135](https://openedx.atlassian.net/browse/OSPR-3135) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by natabene** [comment]
@cclauss Thank you for your contribution. @edx/incr-reviewers Would you like to give this a look?

File: .travis.yml [hunk 1]
```diff
@@ -8,18 +8,18 @@ env:
...
-      env: TOXENV=quality
   exclude:
     - python: 2.7
       env: TOXENV=django20
     - python: 2.7
       env: TOXENV=django21
-  allow_failures:
     - python: 3.6
-    - python: 3.7
+      env: TOXENV=quality
```
Comment (jmbowman) [inline .travis.yml]: Why skip the quality check on 3.6?  That's probably the version we'll be running in production next.

Comment (cclauss) [reply .travis.yml]: In my experience the Abstract Syntax Tree (and thus the linter results) of Python 3.6 and 3.7 are nearly identical with two exceptions where Python 3.7 is _stricter_ that 3.6:
1. The strictness around __StopIteration__ that trips up pylint 1.x
2. __async__ and __await__ are keywords which has tripped up Spark, PyTorch, Zappa, etc. but not edX

My sense is that no linting test will pass on 3.7 but fail on 3.6 but we can swap these if you want.  Or we can test quality on all three just to be certain.

**Comment by edx-webhook** [comment]
@cclauss 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/xblock-utils&entry.867055334=https://github.com/edx/xblock-utils/pull/57&entry.1484655318=https://github.com/cclauss&entry.752974735=2019-02-25+19:40&entry.1917517419=2019-02-26+15:08&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**